nomad/structs/network_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"testing"
 
 	"github.com/hashicorp/nomad/ci"
+	"github.com/stretchr/testify/assert"
```

Comment:
just JYI the top-level `test` package from `shoenig/test` is analogous to `testify/assert` (marking failure without stopping)